### ISSUE TITLE
Mac port

### ISSUE THREAD

**Issue by nullpainter** [mika76/mamesaver#32]
What better way to maintain momentum than to launch straight into the next thing? 😜 

I'm only being _partly_ facetious. I'm a Mac guy at home, I've never done OS X development and there's a dearth of good screensavers out there...

**Comment by mika76** [comment]
Man I've wanted to do a mac port for years! Just never got the motivation to get it going. I'm also a mac guy these days so this would be very very welcome. Only thing is it's really a different project altogether. Were you thinking try in xamarin or go whole hog and do it in swift or objective-c? I'm not sure if xamarin is an option though - some research and prototyping is probably a good first step...

**Comment by nullpainter** [comment]
I'd probably stick with familiar territory - plus I have a vested interest in getting up to speed with Xamarin. I definitely don't want to subject myself to Objective-C, but absolutely agree that a prototype would be prudent. It would naturally be ideal if as much code could be shared between the two projects.

I may create a new project and have a play.

**Comment by antdude** [comment]
Linux port too please. ;)

**Comment by mika76** [comment]
@antdude good idea 😊 I added a new issue for it at #48 - maybe follow that one...

**Comment by antdude** [comment]
Thanks. :)

**Comment by nullpainter** [comment]
@nzcodarnoc FYI, if you're still keen. I will be adding a few changes as part of #47 but these don't change functionality dramatically.

**Comment by mika76** [comment]
@nullpainter I think you commented on the wrong issue?

**Comment by mika76** [comment]
Ah I see 😊

**Comment by nzcodarnoc** [comment]
Yep, still keen, but you know: also quite busy.
